### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by 1anisim**
From Alex.Anisimov [JB]
Added a section about Multiplatform Capabilities, highlighted KMM

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ Welcome to [Kotlin](https://kotlinlang.org/)! Some handy links:
  * [Public Slack channel](https://slack.kotlinlang.org/)
  * [TeamCity CI build](https://teamcity.jetbrains.com/project.html?tab=projectOverview&projectId=Kotlin)
 
+## Kotlin Multiplatform capabilities
+
+Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for [different platforms](https://kotlinlang.org/docs/reference/mpp-supported-platforms.html) while retaining the flexibility and benefits of native programming.
+
+ * [Koltin Multiplatform Mobile](https://kotlinlang.org/lp/mobile/) for sharing code between Android and iOS
```
Comment (abelkov) [inline ReadMe.md:25]: Koltin

File: ReadMe.md [hunk 2]
```diff
@@ -18,6 +18,16 @@ Welcome to [Kotlin](https://kotlinlang.org/)! Some handy links:
...
+ * [Getting Started with Koltin Multiplatform Mobile Guide](https://kotlinlang.org/docs/mobile/create-first-app.html)
```
Comment (abelkov) [inline ReadMe.md:26]: Koltin!

File: ReadMe.md [hunk 3]
```diff
@@ -18,6 +18,16 @@ Welcome to [Kotlin](https://kotlinlang.org/)! Some handy links:
...
+ * [Koltin Multiplatform Benefits](https://kotlinlang.org/docs/reference/multiplatform.html)
```
Comment (abelkov) [inline ReadMe.md:27]: Koltin

**Comment by abelkov** [comment]
Is this information really needed in a Git repository intended for project contributors?

**Comment by 1anisim** [comment]
@abelkov I have sent you a slack message 
Fixed typos

**Comment by udalov** [comment]
> I have sent you a slack message

Can't we discuss it publicly though? I'm also interested in an answer to Alexey's question

**Comment by 1anisim** [comment]
Sure, no problem

I previously discussed this with Stanis Erokhin and he agreed to approve it.

In two words, the situation is as follows:
About 10K new users come from github to Kotlinglang per month. It seems that they are the exact target audience of KMM, and it will be only for the best if we can attract some of them to the KMM portal. 

I am sure that we will not receive any negative feedback from the community, therefore if there is something, then we can always return everything as it was. Technically, GitHub is one of the communication channels where we communicate with the target audience, it seems that we can benefit from using it in a such way.

The block about the multiplatform comes immediately after the introduction, which is also designed for those who want to use Kotlin as a working tool, so it seems that the part about the multiplatform should fit good.

In general, this is part of an experiment of increasing conversions from 3-rd party sites to our portal. Here I want to understand whether ReadMe helps to somehow bring more traffic to our resource and if it does what is the quality it has the and what quality it is. If everything works well for KMM, then it will be possible to advise other projects to think about something similar.

File: ReadMe.md [hunk 4]
```diff
@@ -18,6 +18,16 @@ Welcome to [Kotlin](https://kotlinlang.org/)! Some handy links:
...
  * [TeamCity CI build](https://teamcity.jetbrains.com/project.html?tab=projectOverview&projectId=Kotlin)
 
+## Kotlin Multiplatform capabilities
+
+Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for [different platforms](https://kotlinlang.org/docs/reference/mpp-supported-platforms.html) while retaining the flexibility and benefits of native programming.
+
+ * [Kotlin Multiplatform Mobile](https://kotlinlang.org/lp/mobile/) for sharing code between Android and iOS
+ * [Getting Started with Kotlin Multiplatform Mobile Guide](https://kotlinlang.org/docs/mobile/create-first-app.html)
+ * [Kotlin Multiplatform Benefits](https://kotlinlang.org/docs/reference/multiplatform.html)
+ * [Share code on All Platfroms](https://kotlinlang.org/docs/reference/mpp-share-on-platforms.html#share-code-on-all-platforms)
```
Comment (udalov) [inline ReadMe.md:28]:
```suggestion
 * [Share code on All Platforms](https://kotlinlang.org/docs/reference/mpp-share-on-platforms.html#share-code-on-all-platforms)
```

File: ReadMe.md [hunk 5]
```diff
@@ -18,6 +18,16 @@ Welcome to [Kotlin](https://kotlinlang.org/)! Some handy links:
...
+ * [Share code on similar Platforms](https://kotlinlang.org/docs/reference/mpp-share-on-platforms.html#share-code-on-similar-platforms)
```
Comment (udalov) [inline ReadMe.md:29]:
```suggestion
 * [Share code on similar platforms](https://kotlinlang.org/docs/reference/mpp-share-on-platforms.html#share-code-on-similar-platforms)
```

**Pull request merged**